README.md

```diff
@@ -4,7 +4,32 @@
 [![Build Status](https://travis-ci.org/praekeltfoundation/docker-ci-deploy.svg?branch=develop)](https://travis-ci.org/praekeltfoundation/docker-ci-deploy)
 [![codecov](https://codecov.io/gh/praekeltfoundation/docker-ci-deploy/branch/develop/graph/badge.svg)](https://codecov.io/gh/praekeltfoundation/docker-ci-deploy)
 
-Python script to help push Docker images to a registry using CI services
+A command-line tool to help generate tags and push Docker images to a registry. Simplifies deployment of Docker images from CI services such as Travis CI.
+
+In a single command, `docker-ci-deploy` can:
+* Change the tags on images
+* Add version information to image tags
+* Add registry addresses to image tags
+* Login to a registry
+* Push tags to a registry
+
+The best way to try out `docker-ci-deploy` is to give it a spin with the `--dry-run` flag and observe all the :
+```
+> $ docker-ci-deploy --tag-version 2.7.13 --tag-latest \
+      --registry registry:5000 --login 'janedoe:pa55word' --dry-run \
```

Comment:
Should this be `pa$$word` to match other places in the README?

(I know we switched to `pa55word` in tests to make matching easier, but that's not really relevant here.)